conf/battle/battle.conf

```diff
@@ -46,9 +46,14 @@ damage_walk_delay_rate: 100
 multihit_delay: 80
 
 // Damaged delay rate for players (Note 2)
-// (Setting to no/0  will be like always endure)
+// This affects the damage delay that is sent to the client and is the base value for the walk delay.
 player_damage_delay_rate: 100
 
+// Always have endure? (Note 3)
+// The unit types defined here cannot be stopped by damage.
+// Please note that this makes it impossible to display critical hits on these unit types.
```

Comment:
Maybe briefly explain why it's impossible so people don't start asking about it

Reply:
Improved description.